pkg/apis/componentconfig/v1alpha1/types.go

```diff
@@ -204,6 +204,12 @@ type KubeletConfiguration struct {
 	// default /var/run/kubernetes). If tlsCertFile and tlsPrivateKeyFile
 	// are provided, this flag will be ignored.
 	CertDirectory string `json:"certDirectory"`
+	// BootstrapAuthToken is a string token that is used for getting a x509
+	// certificate from the API server that matches the private key generated
+	// by kubelet.
+	// The result certificate will be stored in tlsCertFile, and the generated
+	// private key file will be stored in tlsPrivateKeyFile.
+	BootstrapAuthToken string `json:"bootstrapAuthToken"`
```

Comment:
Ok, this is really the culprit. I've changed this line to `RequestTLSCert bool `json:"requestTLSCert"`` and re-ran codegen - all looks well (I have `autoConvert_v1alpha1_KubeletConfiguration_To_componentconfig_KubeletConfiguration`  and friends back). Looks like a codegen bug... it should have warned us, don't we think?